kubeflow/trainer/types/types.py

```diff
@@ -231,6 +231,7 @@ class TrainerType(Enum):
 class RuntimeTrainer:
     trainer_type: TrainerType
     framework: str
+    image: str
```

Comment:
@astefanutti @Fiona-Waters I made `image` mandatory for the RuntimeTrainer.
The container should always has an image, but for the local subprocess backend, we can populate some const value there.
What do you think about it ?